package.json

```diff
@@ -21,7 +21,8 @@
   "dependencies": {
     "chalk": "^4.1.2",
     "lit-element": "^2.5.1",
-    "lit-html": "^1.4.1"
+    "lit-html": "^1.4.1",
+    "mark.js": "^8.11.0"
```

Comment:
Please update this to pin this version. With this running an `npm update` will install the 8.11.1 because it's a patch. 

```shell
$ npm i mark.js@8.11.0 --save-exact
```

This will update the package.json and the package-lock.json.